resources/views/purple-chrome.blade.php

```diff
@@ -22,11 +22,17 @@
             @if (config('app.env') !== 'production')
             <span class="badge badge-warning d-none d-sm-inline" style="font-size: large;">
                 <i class="fa fa-exclamation-triangle" aria-hidden="true"></i> Environment: {{ config('app.env') }}
+                @if (auth()->user()->db_schema)
```

Comment:
you are accessing app specific variables, what you are accessing is set for ecats-ui here: https://github.com/NIT-Administrative-Systems/ecats-ui/blob/56c466783516adec669dba5a0e4c4b578e5acc24/app%2FAuth%2FSsoGate.php#L25

You need to precede this with isset() function or this will potentially throw an error on any other application that uses it since auth()->user() may be null, or db_schema may not be set for that variable.

Reply:
There should be an [`@includeIf` directive](https://laravel.com/docs/8.x/blade#including-subviews) that will render a view if the file exists, and otherwise do nothing.

That may be a cleaner way to implement this -- you can do an `@includeIf('layout._badges')`, and add that file to the ecats-ui app?

You could also `@yield('badges')` here and do a badges block somewhere in ecats-ui, but I don't know what the best place would be for that block to live.

Reply:
Thanks for the suggestion @nie7321, I went with the `@includeIf` approach